tools/ControllerFactory.php

```diff
@@ -113,7 +110,14 @@ public function render(string $twigPath, array $data): void {
 		echo $this->twig->render($twigPath, $data);
 	}
 
-	public function redirect($path){
+	/**
+	 * Redirige à l'url demandé.
+	 *
+	 * @param string $path
+	 */
+	public function redirect(string $path): void{
+		header('HTTP/1.1 Moved Permanently', false, 302);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6084628)